src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/HearingRoute.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import com.fasterxml.jackson.annotation.JsonEnumDefaultValue;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+
+@Getter
+public enum HearingRoute {
+
+    @JsonEnumDefaultValue
+    @JsonProperty
+    LIST_ASSIST("listAssist"),
+    @JsonProperty
+    GAPS("gaps");
+
+    private final String type;
+
+    HearingRoute(String type) {
```

Comment:
Optional: Can use lombok `@RequiredArgsConstructor` for this

Reply:
Yup that is fine will update those